README.md

```diff
@@ -60,12 +60,12 @@ On Ubuntu >= 18.04 (for example):
 
 ```sh
 sudo apt-get install --yes                                                       \
-            autoconf bison clang-8 cmake curl flex gcc jq libboost-test-dev      \
+            autoconf bison clang-10 cmake curl flex gcc jq libboost-test-dev     \
             libcrypto++-dev libffi-dev libgflags-dev libjemalloc-dev libmpfr-dev \
             libprocps-dev libsecp256k1-dev libssl-dev libtool libyaml-dev        \
-            lld-8 llvm-8-tools make maven netcat-openbsd openjdk-11-jdk          \
-            pkg-config python3 python-pygments python-recommonmark               \
-            python-sphinx rapidjson-dev time z3 zlib1g-dev
+            lld-10 llvm-10-tools make maven netcat-openbsd openjdk-11-jdk        \
+            pkg-config python3 python-pygments python-sphinx                     \
+            rapidjson-dev time z3 zlib1g-dev
```

Comment:
```suggestion
            pkg-config python3 python-pygments rapidjson-dev                     \
            time z3 zlib1g-dev
```

Reply:
This one was marked resolved, but I thought you meant to get rid of `python-sphinx`? It's still there.